service/controller/app/v1/resource/configmap/current.go

```diff
@@ -20,6 +21,31 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
+	ns, err := r.k8sClient.CoreV1().Namespaces().Get(key.Namespace(cr), metav1.GetOptions{})
```

Comment:
Whenever cluster got deleted, we will cancel all reconciliation since network issue in app-operator, For the next loop in reconciliation, we would start from `configmap` resource. 
So I think checking the namespace condition only in `configmap` makes sense, WDYT?